app/src/main/java/com/jmperezra/avilaentapas/features/main/MainActivity.kt

```diff
@@ -2,11 +2,56 @@ package com.jmperezra.avilaentapas.features.main
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.MenuItem
+import androidx.fragment.app.Fragment
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import com.google.android.material.navigation.NavigationView
 import com.jmperezra.avilaentapas.R
+import com.jmperezra.avilaentapas.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+
+    private val binding:ActivityMainBinding by lazy {
+        ActivityMainBinding.inflate(layoutInflater)
+    }
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setContentView(binding.root)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        return when (item.itemId) {
+            R.id.action_tapas -> {
+                // Respond to navigation item 1 click
+                //replaveFragment(container_fragment, Fra)
+                true
+            }
+            R.id.action_competition -> {
+                // Respond to navigation item 2 click
+                //replaveFragment(container_fragment, )
+                true
+            }
+            R.id.action_event -> {
+                // Respond to navigation item 3 click
+                //replaveFragment(container_fragment)
+                true
+            }
+            R.id.action_my_profile -> {
+                // Respond to navigation item 4 click
+                //replaveFragment()
+                true
+            }
+            else -> super.onOptionsItemSelected(item)
+        }
     }
-}
\ No newline at end of file
+
+
+    private fun replaceFragment(fragment: Fragment) {
+        val idLayout=R.id.container_fragment
+        val fragmentTransition = supportFragmentManager.beginTransaction()
+        fragmentTransition.replace(idLayout, fragment)
```

Comment:
Tenemos que evitar crearnos variables en estos casos:
```suggestion
        fragmentTransition.replace(R.id.container_fragment, fragment)
```

Reply:
solucionado